src/commands/classes/Contest.command.ts

```diff
@@ -1,43 +1,169 @@
 import Command from "./Command";
-import { Message } from "discord.js";
+import { Message, Channel, TextChannel } from "discord.js";
```

Comment:
Strings should all use single quotes rather than double quotes. This goes for every file in the project, unless of course its a JSON object